skafka/src/main/scala/com/evolutiongaming/skafka/producer/ProducerRecord.scala

```diff
@@ -17,9 +17,15 @@ final case class ProducerRecord[+K, +V](
 
 object ProducerRecord {
 
-  def apply[K, V](topic: Topic, value: V, key: K): ProducerRecord[K, V] = {
-    ProducerRecord(topic = topic, value = Some(value), key = Some(key))
-  }
+  def apply[K, V](topic: Topic, value: V, key: K): ProducerRecord[K, V] =
+    ProducerRecord(
+      topic = topic, 
+      value = Some(value), 
+      key = Some(key), 
+      partition = None, 
```

Comment:
Did the compiler complain here?

Reply:
Yes, it couldn't find the appropriate `apply` method. The easiest solution was to pass all arguments explicitly.